### PR TITLE
deploy: automatically deploy sdist and wheel builds to PyPI.

### PR DESCRIPTION
**Pull request by beardypig**
Deploys using a new Streamlink user on PyPI, I have emailed the credentials to @cdrage and @gravyboat and already added the credentials to travis.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -63,6 +63,13 @@ deploy:
     on:
       branch: master
       condition: $BUILD_INSTALLER == yes && $TRAVIS_EVENT_TYPE == cron
+  - provider: pypi
+    user: "${PYPI_USER}"
+    password: "${PYPI_PASSWORD}"
+    distributions: "sdist bdist_wheel"
+    on:
+      tags: true
+      condition: $DEPLOY_PYPI == yes
```
Comment (back-to) [inline .travis.yml]: `repo: streamlink/streamlink`
https://docs.travis-ci.com/user/conditional-builds-stages-jobs/

on every deploy provider would be nice,

currently `git push origin 0.12.1` on a **fork** with travis-ci will fail

https://travis-ci.org/back-to/streamlink/jobs/382711711#L2592

---

and all the `PyPI` code from https://github.com/streamlink/streamlink/blob/master/script/release.sh
can be removed

Comment (beardypig) [reply .travis.yml]: Yeah, still need to update the release script - I'm hoping it can be removed soon :)

**Comment by gravyboat** [comment]
I got your email, thanks @beardypig.

**Comment by cdrage** [comment]
I confirm that I got your e-mail too @beardypig :+1:

File: .travis.yml (hunk 1 above)
Comment (beardypig) [reply .travis.yml]: I added the `repo` condition :)

**Comment by gravyboat** [comment]
Any other feedback on this? If not I'm going to merge it. Changes look good to me.

**Comment by beardypig** [comment]
I have combined the changes in to #1701, as they are closely related and can be merged in to a single script.

**Comment by gravyboat** [comment]
Okay thanks for the heads up @beardypig, closing.

**Pull request closed without merging**